src/main/scala/swoop/modelo/Bob.scala

```diff
@@ -0,0 +1,29 @@
+package swoop.modelo
+
+import scala.language.dynamics
+
+case class ModeloBobException(smth: String)  extends Exception(smth)
+
+case class Bob(
```

Comment:
Why is this called "Bob"? I don't get the name.

Reply:
![image](https://user-images.githubusercontent.com/2722395/116113200-48ac1e00-a686-11eb-87c3-4fc3cf531bf4.png)